config/routes.rb

```diff
@@ -64,10 +64,11 @@
   get 'tags/:slug/feed',         to: 'tags#feed', defaults: { format: 'atom' }, as: :tag_feed
 
   # Podcast
-  get 'podcast/feed',             to: 'podcast#feed',       as: :podcast_feed, defaults: { format: 'rss' }
-  get 'podcast',                  to: 'podcast#index',      as: :podcast
-  get 'podcast/:slug',            to: 'podcast#show',       as: :episode
-  get 'podcast/:slug/transcript', to: 'podcast#transcript', as: :episode_transcript
+  get 'podcast/feed',             to: 'podcast#feed',        as: :podcast_feed, defaults: { format: 'rss' }
+  get 'podcast',                  to: 'podcast#index',       as: :podcast
+  get 'podcast/:slug',            to: 'podcast#show',        as: :podcast_detail
+  get 'podcast/:slug/episodes/:id', to: 'episodes#show',     as: :podcast_episode
+  get 'podcast/:slug/transcript', to: 'episodes#transcript', as: :episode_transcript
```

Comment:
```suggestion
  get 'podcast/:slug/episodes/:id/transcript', to: 'episodes#transcript', as: :episode_transcript
```

A transcript needs to hang off of a single episode.